docs/_components/accordions.md

```diff
@@ -7,129 +7,8 @@ lead: Accordions are a list of headers that can be clicked to hide or reveal add
-    <li>
-      <button class="usa-accordion-button"
-        aria-expanded="true" aria-controls="amendment-b-1">
-        First Amendment
-      </button>
-      <div id="amendment-b-1" class="usa-accordion-content">
-        <p>
-        Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances.
-        </p>
-      </div>
-    </li>
-    <li>
-      <button class="usa-accordion-button"
-        aria-controls="amendment-b-2">
-        Second Amendment
-      </button>
-      <div id="amendment-b-2" class="usa-accordion-content">
-        <p>
-        A well regulated Militia, being necessary to the security of a free State, the right of the people to keep and bear Arms, shall not be infringed.
-        </p>
-      </div>
-    </li>
-    <li>
-      <button class="usa-accordion-button"
-          aria-controls="amendment-b-3">
-        Third Amendment
-      </button>
-      <div id="amendment-b-3" class="usa-accordion-content">
-        <p>
-        No Soldier shall, in time of peace be quartered in any house, without the consent of the Owner, nor in time of war, but in a manner to be prescribed by law.
-        </p>
-      </div>
-    </li>
-    <li>
-      <button class="usa-accordion-button"
-        aria-controls="amendment-b-4">
-        Fourth Amendment
-      </button>
-      <div id="amendment-b-4" class="usa-accordion-content">
-        <p>
-        The right of the people to be secure in their persons, houses, papers, and effects, against unreasonable searches and seizures, shall not be violated, and no Warrants shall issue, but upon probable cause, supported by Oath or affirmation, and particularly describing the place to be searched, and the persons or things to be seized.
-        </p>
-      </div>
-    </li>
-    <li>
-      <button class="usa-accordion-button"
-        aria-controls="amendment-b-5">
-        Fifth Amendment
-      </button>
-      <div id="amendment-b-5" class="usa-accordion-content">
-        <p>
-        No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.
-        </p>
-      </div>
-    </li>
-  </ul>
-</div>
-
+{% include code/preview.html component="preview-accordion" %}
+{% include code/accordion.html component="code-accordion" %}
```

Comment:
These files exist in the repository, but the file names represented by the `component=""` parameter are generated using gulp.